### ISSUE TITLE
[Bug]: Impossible to get rid of seams with ultimate SD upscaler script

### ISSUE THREAD

**Issue by Aresic** [lshqqytiger/stable-diffusion-webui-directml#113]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

Hello, I tried every options and I cant get rid of the seams on my image maybe im doing something wrong but I read the FAQ and im stiill not able to fix this.

I'm running on a AMD 6800xt GPU.

### Steps to reproduce the problem

I first posted on the wrong repository so here are my screenshots

#https://github.com/Coyote-A/ultimate-upscale-for-automatic1111/issues/78

Original image I want to upscale is : 

![00014-250319910](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/132944008/14cf74ed-14b7-466a-ae81-e1dcd04247f8)

### What should have happened?

it seems that if I use denoising at 1 as a test only the top part of my image is being changed . 

Lets say I want to get a 2x upscale on a 608*760 image. 
So I use a tile of 512 ( I tried other size dont change the problem) in the script and no matter what other settings I use , seam fix options padding mask blur and all .. it will do something like this : 

![00000-250319910 0](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/132944008/696e3df5-4f64-4952-8be9-a95f865908d4)

So you can see that obviously the tiles only worket in the top of the image. 

Once that said best result I cant get  is something like this : 

![00048-250319910 0](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/132944008/a7e1af3e-fe7e-45b7-b61f-f0a886aa5b63)

Then I use photoshop to clean the image

### Commit where the problem happens

https://google.com/search?q=62c5a822ec7149eed56c23b10effa88a7edc4db8a72990ef0d96179890320d81

### What platforms do you use to access the UI ?

Windows

### What browsers do you use to access the UI ?

Google Chrome, Microsoft Edge

### Command Line Arguments

```Shell
I do use :

@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS=--medvram --no-half-vae --opt-split-attention --upcast-sampling --disable-nan-check --autolaunch
set SAFETENSORS_FAST_GPU=1

call webui.bat
```

### List of extensions

<html>
<body>

Extension | URL | Version | Update
-- | -- | -- | --
LyCORIS | https://github.com/KohakuBlueleaf/LyCORIS | ded70eed (Sun Apr 16 15:51:01 2023) | unknown
Stable-Diffusion-Webui-Civitai-Helper | https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper | fcf5a27a (Fri Apr 14 05:39:01 2023) | unknown
a1111-sd-webui-lycoris | https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris | 514511d7 (Thu May 4 16:31:40 2023) | unknown
batch-face-swap |   |   | unknown
multidiffusion-upscaler-for-automatic1111 | https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111.git | 97fcc2ce (Mon May 8 07:34:56 2023) | unknown
sd-dynamic-prompts | https://github.com/adieyal/sd-dynamic-prompts.git | a53adac9 (Sun Apr 23 13:28:49 2023) | unknown
sd-webui-additional-networks | https://github.com/kohya-ss/sd-webui-additional-networks.git | c6006f8e (Mon May 8 13:05:50 2023) | unknown
sd-webui-controlnet | https://github.com/Mikubill/sd-webui-controlnet | c9c8ca6e (Sun May 7 17:15:00 2023) | unknown
sd_webui_stealth_pnginfo | https://github.com/ashen-sensored/sd_webui_stealth_pnginfo | 63d6e051 (Mon Apr 24 19:19:41 2023) | unknown
stable-diffusion-webui-images-browser |   |   | unknown
ultimate-upscale-for-automatic1111 | https://github.com/Coyote-A/ultimate-upscale-for-automatic1111 | 756bb505 (Fri May 5 00:22:21 2023) | unknown
LDSR | built-in |   |  
Lora | built-in |   |  
ScuNET | built-in |   |  
SwinIR | built-in |   |  
prompt-bracket-checker | built-in |   |  

</body>
</html>Extension	URL	Version	Update
LyCORIS	https://github.com/KohakuBlueleaf/LyCORIS	ded70eed (Sun Apr 16 15:51:01 2023)	unknown
Stable-Diffusion-Webui-Civitai-Helper	https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper	fcf5a27a (Fri Apr 14 05:39:01 2023)	unknown
a1111-sd-webui-lycoris	https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris	514511d7 (Thu May 4 16:31:40 2023)	unknown
batch-face-swap			unknown
multidiffusion-upscaler-for-automatic1111	https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111.git	97fcc2ce (Mon May 8 07:34:56 2023)	unknown
sd-dynamic-prompts	https://github.com/adieyal/sd-dynamic-prompts.git	a53adac9 (Sun Apr 23 13:28:49 2023)	unknown
sd-webui-additional-networks	https://github.com/kohya-ss/sd-webui-additional-networks.git	c6006f8e (Mon May 8 13:05:50 2023)	unknown
sd-webui-controlnet	https://github.com/Mikubill/sd-webui-controlnet	c9c8ca6e (Sun May 7 17:15:00 2023)	unknown
sd_webui_stealth_pnginfo	https://github.com/ashen-sensored/sd_webui_stealth_pnginfo	63d6e051 (Mon Apr 24 19:19:41 2023)	unknown
stable-diffusion-webui-images-browser			unknown
ultimate-upscale-for-automatic1111	https://github.com/Coyote-A/ultimate-upscale-for-automatic1111	756bb505 (Fri May 5 00:22:21 2023)	unknown
LDSR	[built-in](http://127.0.0.1:7860/)		
Lora	[built-in](http://127.0.0.1:7860/)		
ScuNET	[built-in](http://127.0.0.1:7860/)		
SwinIR	[built-in](http://127.0.0.1:7860/)		
prompt-bracket-checker	[built-in](http://127.0.0.1:7860/)

### Console logs

```Shell
venv "C:\Stable\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: <none>
Installing requirements for Web UI
Installing sd-dynamic-prompts requirements.txt



Launching Web UI with arguments: --medvram --no-half-vae --opt-split-attention --upcast-sampling --disable-nan-check --autolaunch
Interrogations are fallen back to cpu. This doesn't affect on image generation. But if you want to use interrogate (CLIP or DeepBooru), check out this issue: https://github.com/lshqqytiger/stable-diffusion-webui-directml/issues/10
Warning: caught exception 'Torch not compiled with CUDA enabled', memory monitor disabled
No module 'xformers'. Proceeding without it.
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: C:\Stable\stable-diffusion-webui-directml\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
Civitai Helper: No setting file, use default
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
ControlNet v1.1.150
ControlNet v1.1.150
Loading weights [62c5a822ec] from C:\Stable\stable-diffusion-webui-directml\models\Stable-diffusion\deimosMix_aeV01.safetensors
Creating model from config: C:\Stable\stable-diffusion-webui-directml\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: C:\Stable\stable-diffusion-webui-directml\models\VAE\vae-ft-mse-840000-ema-pruned.vae.pt
Applying cross attention optimization (Doggettx).
Textual inversion embeddings loaded(14): achechik, bad-artist-anime, bad-artist, bad-hands-5, bad_prompt, bad_prompt_version2, cmack, easynegative, jg_4mouranth, ng_deepnegative_v1_75t, ohmyfrick_yogapuss, pjensen, stoya-v2, ulzzang-6500-v1.1
Model loaded in 6.1s (load weights from disk: 0.2s, create model: 0.4s, apply weights to model: 4.1s, apply half(): 0.6s, load VAE: 0.8s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 14.7s (import torch: 3.5s, import gradio: 0.6s, import ldm: 0.5s, other imports: 0.9s, load scripts: 1.2s, load SD checkpoint: 6.2s, create ui: 1.5s, gradio launch: 0.2s).
Canva size: 1216x1536
Image size: 608x760
Scale factor: 3
Upscaling iteration 1 with scale factor 3
        Tile 1/20
        Tile 2/20
        Tile 3/20
        Tile 4/20
        Tile 5/20
        Tile 6/20
        Tile 7/20
        Tile 8/20
        Tile 9/20
        Tile 10/20
        Tile 11/20
        Tile 12/20
        Tile 13/20
        Tile 14/20
        Tile 15/20
        Tile 16/20
        Tile 17/20
        Tile 18/20
        Tile 19/20
        Tile 20/20
Tile size: 512x512
Tiles amount: 9
Grid: 3x3
Redraw enabled: True
Seams fix mode: NONE
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:18<00:00,  1.07it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.17it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.17it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.17it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.17it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:16<00:00,  1.18it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:16<00:00,  1.18it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.17it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:17<00:00,  1.16it/s]
Total progress: 100%|████████████████████████████████████████████████████████████████| 180/180 [02:50<00:00,  1.06it/s]
Total progress: 100%|████████████████████████████████████████████████████████████████| 180/180 [02:50<00:00,  1.19it/s]
```

**Comment by gigend** [comment]
same issue with me, only produce top tile image (even if the denoising strength is changed).
